brownie-config.yaml

```diff
@@ -6,19 +6,18 @@ autofetch_sources: true
 
 # require OpenZepplin Contracts
 dependencies:
-  - OpenZeppelin/openzeppelin-contracts@3.1.0
+  - OpenZeppelin/openzeppelin-contracts@4.4.1
```

Comment:
what's the release history look like here? any noteworthy changes that impact us?

Reply:
I have tried to get an idea of the changes logs for every versions https://github.com/OpenZeppelin/openzeppelin-contracts/releases
 
v3.2.0 -> add proxy to the project
 v3.3.0 -> solidity 0.7
 v3.4.0 -> fix ERC777 potential reentrancy
v4.0.0  -> Solidity 0.8
 v4.1.0 -> UUPS proxies as a cheaper alternative to Transparent proxies
 v4.2.0 -> ERC20Votes
 v4.3.0 -> More governance stuff
 v4.4.0 -> Many small changes

Reply:
Just make sure you've explored all of our dependencies and dependency tree to ensure we are getting rugged on a tiny change that doesn't make sense or affects our current utilization.

Reply:
I don't see any solidity dependencies on https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/package.json that shouldn't be an issue.